cli/src/semgrep/commands/ci.py

```diff
@@ -308,8 +327,11 @@ def ci(
     if dataflow_traces is None:
         dataflow_traces = engine_type.has_dataflow_traces
 
-    console.print(Title("Scan Environment", order=2))
-    console.print(Padding(scan_env, (0, 2)))
+    if max_memory is None:
+        max_memory = engine_type.default_max_memory
+
+    if interfile_timeout is None:
+        interfile_timeout = engine_type.default_interfile_timeout
```

Comment:
didn't you fix this in a different way in the other PR

Reply:
lol

<img width="685" alt="CleanShot 2023-03-02 at 00 07 50@2x" src="https://user-images.githubusercontent.com/1060436/222336499-c5c0a095-18b8-4d50-8e8f-57c3f6b56fa7.png">